AdGoBye/appsettings.json

```diff
@@ -11,7 +11,9 @@
         "DisableBackupFile": false,
         "BlocklistURLs": [],
         "DisablePluginInstallWarning": false,
-        "EnableRecompression":  false
+        "EnableRecompression":  false,
+        "MaxIndexerThreads": 16,
+        "MaxPatchThreads": 16
```

Comment:
As per IM, I think the thread number is fine, any number we pick will be rather [xkcd 221-ish](https://xkcd.com/221) since we don't have any stats on our users to make a good default baseline short of using processor count.

Let's leave it in the ballpark of the users on how aggressive they want it.
If we end up documenting the Options at some point (which we might have to with how they're growing), we should probably document that [`-1` allows for unlimited parallelism](https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.paralleloptions.maxdegreeofparallelism?view=net-8.0#remarks).